cfe_ts_crc.c

```diff
@@ -102,20 +102,21 @@ uint32 CalculateCRC(void *DataPtr, uint32 DataLength, uint32 InputCRC)
 
 int main(int argc, char **argv)
 {
-    int    readSize;
-    int    skipSize = 0;
-    int    fileSize = 0;
-    uint32 fileCRC  = 0;
-    int    fd;
-    int    done = 0;
-    char   buffer[100];
+    int     readSize;
+    int     skipSize = 0;
+    int     fileSize = 0;
+    uint32  fileCRC  = 0;
+    int     fd;
+    int     done = 0;
+    char    buffer[100];
+    __off_t offsetReturn = 0;
```

Comment:
This should be just `off_t` - not `__off_t`  (the former is the posix type, the latter is the internal implementation type that the C library uses)